Makefile

```diff
@@ -203,8 +207,27 @@ test-submit:      ## Submit test training job
 				kubectl get pods | grep learner- | awk '{print $$1}' | xargs -I '{}' kubectl logs '{}' -c load-data; \
 				exit 1);
 
+pull-dockerhub-images:  ## Pull FfDL images from dockerhub
+pull-dockerhub-images: $(addprefix pull-, $(TEST_IMAGES))
+
+$(addprefix pull-, $(TEST_IMAGES)): pull-%: %
+	@TRAVIS_IMAGES=$< make .pull-dockerhub-images
```

Comment:
nitpick: perhaps better `TRAVIS_IMAGE` instead of `TRAVIS_IMAGES`? (also in lines 221, 226, 229)